src/internal/methods/ESP/ESP32/NeoEsp32RmtXMethod.h

```diff
@@ -76,6 +76,7 @@ template<typename T_SPEED, typename T_INVERTED = NeoEsp32RmtNotInverted> class N
         // wait until the last send finishes before destructing everything
         // arbitrary time out of 10 seconds
 
+        ESP_ERROR_CHECK(rmt_disable(_channel)); // TroyHacks: added to disable the channel before deleting it
```

Comment:
This should be just before the rmt_del_channel.

Don't want to disable a channel that is still actively sending otherwise it could leave the LEDs in a poor state.